Sources/Public/CalendarView.swift

```diff
@@ -804,6 +810,22 @@ public final class CalendarView: UIView {
     }
   }
 
+  /// This is needed to ensure that the `subviews` array is sorted according to each sublayer's `zPosition`. This prevents
+  /// z-index-related rendering issues when a `CalendarView` is being snapshotted via `CALayer.render(in:)`.
+  private func sortSubviewsByLayerZPositions() {
+    // Bubble Sort implementation, which makes doing in-place swaps (subview exchanges) easy.
+    for i in 0..<scrollView.subviews.count {
+      for j in 0..<(scrollView.subviews.count - i - 1) {
+        let lhsZPosition = scrollView.subviews[j].layer.zPosition
+        let rhsZPosition = scrollView.subviews[j + 1].layer.zPosition
+
+        if lhsZPosition > rhsZPosition {
+          scrollView.exchangeSubview(at: j, withSubviewAt: j + 1)
+        }
+      }
+    }
+  }
```

Comment:
So yeah, I could add this as a generic internal internal bubble sort implementation and write unit tests for it... but I kind of don't want to bother 😛

I'm not even sure this is needed long term. I need to profile how expensive it is, but I might be able to just replace my calls to `scrollView.addSubview(view)` with calls to `scrollView.insertSubview(view, at: correctIndex)`. If I can do this, then I don't need to manually set the `zPosition` on sublayers. My main concern with this approach is that I'm doing a lot of subview array insert operations (rather than append operations), which I suspect is more expensive, especially since this is done in a hot codepath while scrolling / views are being reused.

We need a quick fix now for some Airbnb stuff, and this code only ever runs if the layer is snappshotted via `render(in:)`, so trying to keep this lightweight.